projects/tinkerbell/hook/Makefile

```diff
@@ -52,7 +52,7 @@ s3-artifacts: $(CREATE_HOOK_FILES)
 $(CREATE_HOOK_FILES): tarballs
 # Modify the linuxkit config file hook/hook.yaml to point to IMAGE_REPO.
 # Upstream make target `dist` is triggered to perform linuxkit build and generate OSIE files ().
-	source $(BUILD_LIB)/common.sh && build::common::use_go_version "1.16" && make dist IMAGE_REPO=$(IMAGE_REPO) LATEST_TAG=$(LATEST_TAG) -C $(REPO)
+	source $(BUILD_LIB)/common.sh && build::common::use_go_version "1.19" && make dist IMAGE_REPO=$(IMAGE_REPO) LATEST_TAG=$(LATEST_TAG) -C $(REPO)
```

Comment:
do we need to update linuxkit in the builder base? if not, can we build it with a newer golang?

Reply:
Yup we updated the version to what upstream hook is tracking.